runtime/sema/path_type.go

```diff
@@ -30,10 +30,6 @@ var PathType = &SimpleType{
 	Comparable:    false,
 	Exportable:    true,
 	Importable:    true,
-	IsSuperTypeOf: func(subType Type) bool {
-		return IsSubType(subType, StoragePathType) ||
-			IsSubType(subType, CapabilityPathType)
-	},
```

Comment:
I had to move this to `checkSubTypeWithoutEquality` to break the init cycle since otherwise there was a cycle: 

```
PathType -> CapabilityPathType -> IsSubType -> checkSubTypeWithoutEquality -> IsHashableStructType -> PathType
```